spinalcordtoolbox/image.py

```diff
@@ -970,26 +970,15 @@ def get_dimension(im_file, verbose=1):
     :param: im_file: Image or nibabel object
     :return: nx, ny, nz, nt, px, py, pz, pt
     """
-    # initialization
-    nx, ny, nz, nt, px, py, pz, pt = 1, 1, 1, 1, 1, 1, 1, 1
-    if type(im_file) is nib.nifti1.Nifti1Image:
+    if isinstance(im_file, (nib.nifti1.Nifti1Image, Image)):
```

Comment:
I see, this works because `Image.header` is a synonym for `Image.hdr`:
https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/9d0dcb070138ce5d71a7987e1bc072d7d778b4c6/spinalcordtoolbox/image.py#L364-L370

And `isinstance` is better than `type(...) is ...`, so that's good.